apps/blog/forms.py

```
@@ -0,0 +1,13 @@
+
+from django import forms
+
+from apps.blog.models import BlogComment
+
+
+class BlogCommentForm(forms.ModelForm):
+    """Capture and validate a public comment for a blog post."""
+
+    class Meta:
+        model = BlogComment
+        fields = ["author_name", "author_email", "body"]
```

Ruff (0.15.1)

[warning] Mutable default value for class attribute

(RUF012)